static/app/views/projectInstall/createProject.tsx

```diff
@@ -149,6 +149,23 @@ function CreateProject() {
 
         ProjectsStore.onCreateSuccess(projectData, organization.slug);
 
+        if (team) {
+          addSuccessMessage(
+            tct('Created project [project]', {
+              project: `${projectData.slug}`,
+              organization: organization.slug,
+            })
+          );
+        } else {
+          addSuccessMessage(
+            tct('Created project [project] under new team [team]', {
+              project: `${projectData.slug}`,
+              team: `#${projectData.team_slug}`,
```

Comment:
why is there a # before team slug and not project slug? Also why is there a # next to project slug in line 179

Reply:
Only the team slug should have a `#` b/c that's how teams show up in the UI. Line 179 should not have the `#`, that was my mistake